spec/lib/process_balancer/manager_spec.rb

```diff
@@ -15,6 +15,11 @@
 
   processes_key = ProcessBalancer::PROCESSES_KEY
 
+  def add_worker(identifier, stale: false)
+    redis.rpush(ProcessBalancer::PROCESSES_KEY, identifier)
+    redis.hmset(identifier, 'beat', Time.now.to_f) unless stale
+  end
```

Comment:
WDYT about this change?
```ruby
def add_worker(*identifiers, stale: false)
  identifiers.each do |id|
    redis.rpush(ProcessBalancer::PROCESSES_KEY, id)
    redis.hmset(id, 'beat', Time.now.to_f) unless stale
  end
end
```

So that later you can run `add_worker('worker 1', 'worker 2')` instead of calling it multiple times.